app/views/adopted/index.csv.erb

```diff
@@ -0,0 +1,5 @@
+<%- headers = ['Lat', 'Lng', 'City ID'] -%>
```

Comment:
Is this used anywhere? It didn't appear to be.

Reply:
No it's not. I removed CSV rendering, and the API allows for JSON only.